addon-test-support/setup-context.js

```diff
@@ -0,0 +1,40 @@
+import { run } from '@ember/runloop';
+import { set, setProperties, get, getProperties } from '@ember/object';
+import buildOwner from './build-owner';
+
+/*
+ * Responsible for:
+ *
+ * * create an owner object and set it on the provided context (e.g. this.owner)
+ * * setup this.set, this.setProperties, this.get, and this.getProperties to the provided context
```

Comment:
might want to use `-` here instead of `*` since the comment block itself already uses `*` 😉